eng/scripts/Fetch-PackageList-Javadoc.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+  .SYNOPSIS
+  Fetch the package list from javadoc jar and checkin to docs/metadata
+
+  .DESCRIPTION
+  The Verify-Links.ps1 script will check whether the files contain any broken links.
+
+  .PARAMETER JavaDocJarLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER DocRepoLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER ArtifactName
+  Specifies the file that contains a set of links to ignore when verifying.
+#>
+[CmdletBinding()]
+param (
+  [string] $JavaDocJarLocation = "",
+  [string] $DocRepoLocation = "",
+  [string] $ArtifactName = ""
+)
+. (Join-Path $PSScriptRoot ../common/scripts/common.ps1)
+
+function Fetch-Namespaces-From-Javadoc ($jarFilePath, $tempLocation, $destination) {
+  Push-Location $tempLocation
+  jar xf $jarFilePath
```

Comment:
What happens if `$tempLocation` is not empty? It's probably OK as long as the files get overwritten but it might make sense to do something like

```pwsh
Remove-Item * -Recurse -Force
```

In the folder before unpacking the jar.

Reply:
Exactly, we need to cleanup folder before unjar otherwise element-list will override.